wix-embedded-mysql/pom.xml

```diff
@@ -86,7 +77,13 @@
             <groupId>mysql</groupId>
             <artifactId>mysql-connector-java</artifactId>
             <version>${mysql.connector.version}</version>
-            <scope>provided</scope>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-dbcp2</artifactId>
```

Comment:
Ok, it's not that clear if you are asking about mysql-connector-java or commons-dbcp2.

I changed commons-dbcp2 to be of test-scope. It's used for testing in creating DataSource.